core/cli_entry.py

```diff
@@ -22,24 +19,26 @@ def upload_run_single_entry(directory, force_upload=False):
 
     :param directory: Directory of the sequencing run to upload
     :param force_upload: When set to true, the upload status file will be ignored and file will attempt to be uploaded
-    :return:
+    :return: ExitReturn
```

Comment:
I created a new `ExitReturn` object that allows us to pass an error message along with the success or fail status to the CLI or GUI, so that the GUI does not have to heavily depend on the CLI core code.